pkgs/top-level/python-packages.nix

```diff
@@ -25797,4 +25797,21 @@ in modules // {
     };
   };
 
+  topydo = buildPythonPackage rec {
+    name = "topydo-${version}";
+    version = "0.9";
+    disabled = (!isPy3k);
+
+    src = pkgs.fetchFromGitHub {
+      owner = "bram85";
+      repo = "topydo";
+      rev = version;
+      sha256 = "0vmfr2cxn3r5zc0c4q3a94xy1r0cv177b9zrm9hkkjcmhgq42s3h";
+    };
+
+    propagatedBuildInputs = with self; [ arrow icalendar ];
+
+    doCheck = false;
```

Comment:
Expressions for packages should be complete. Disabling tests is really a last resort. So yes, all its dependencies should be included. That way it becomes easier for others as well to check whether tests can be enabled.
So please do update this PR with all the dependencies.